File: Java/Booking/BasketItem.java
package LTD_API;

import java.util.Date;
import java.util.List;

public class BasketItem
{
	private int basketItemId;
	private int eventId;
	private String eventName;
	private String imageUrl;
	private Date performanceDate;
	private String areaName;
	private List<String> seats;
	private List<TicketInfoBooking> tickets;
	private int venueId;
	private String venueName;
	private String venueAddress;
	private String venueCity;
	private String venuePostCode;
	private decimal priceSubTotal;
	private int reservationValidity;
	private Date expirationDate;
	
	public int getBasketItemId()
    {
        return basketItemId;
    }
    public void setBasketItemId(int basketItemId)
    {
        this.basketItemId = basketItemId;
    }	
	
	public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }
	
	public String getEventName()
    {
        return eventName;
    }
    public void setEventName(String eventName)
    {
        this.eventName = eventName;
    }
	
	public String getImageUrl()
    {
        return imageUrl;
    }
    public void setImageUrl(String imageUrl)
    {
        this.imageUrl = imageUrl;
    }	
	
	public Date getPerformanceDate()
    {
        return performanceDate;
    }
    public void setPerformanceDate(Date performanceDate)
    {
        this.performanceDate = performanceDate;
    }
	
	public String getAreaName()
    {
        return areaName;
    }
    public void setAreaName(String areaName)
    {
        this.areaName = areaName;
    }
	
	public List<String> getSeats()
    {
        return seats;
    }
    public void setSeats(List<String> seats)
    {
        this.seats = seats;
    }
	
	public List<TicketInfoBooking> getTickets()
    {
        return tickets;
    }
    public void setTickets(List<TicketInfoBooking> tickets)
    {
        this.tickets = tickets;
    }
	
	public int getVenueId()
    {
        return venueId;
    }
    public void setVenueId(int venueId)
    {
        this.venueId = venueId;
    }
	
	public String getVenueName()
    {
        return venueName;
    }
    public void setVenueName(String venueName)
    {
        this.venueName = venueName;
    }
	
	public String getVenueAddress()
    {
        return venueAddress;
    }
    public void setVenueAddress(String venueAddress)
    {
        this.venueAddress = venueAddress;
    }
	
	public String getVenueCity()
    {
        return venueCity;
    }
    public void setVenueCity(String venueCity)
    {
        this.venueCity = venueCity;
    }
	
	public String getVenuePostCode()
    {
        return venuePostCode;
    }
    public void setVenuePostCode(String venuePostCode)
    {
        this.venuePostCode = venuePostCode;
    }
	
	public decimal getPriceSubTotal()
    {
        return priceSubTotal;
    }
    public void setPriceSubTotal(decimal priceSubTotal)
    {
        this.priceSubTotal = priceSubTotal;
    }
	
	public int getReservationValidity()
    {
        return reservationValidity;
    }
    public void setReservationValidity(int reservationValidity)
    {
        this.reservationValidity = reservationValidity;
    }
	
	public Date getExpirationDate()
    {
        return expirationDate;
    }
    public void setExpirationDate(Date expirationDate)
    {
        this.expirationDate = expirationDate;
    }
}

File: PHP/Booking/BasketItem.php
<?php
namespace LTD_API;

class BasketItem
{
    private $basketItemId;
    private $eventId;
    private $eventName;
    private $imageUrl;
    //date format
    private $performanceDate;
    private $areaName;
    //array of String
    private $seats;
    //array of TicketInfoBooking
    private $tickets;
    private $venueId;
    private $venueName;
    private $venueAddress;
    private $venueCity;
    private $venuePostCode;
    private $priceSubTotal;
    private $reservationValidity;
    //date format
    private $expirationDate;
    
    public function getBasketItemId()
    {
        return $this->basketItemId;
    }
    public function setBasketItemId($basketItemId)
    {
        $this->basketItemId = $basketItemId;
    }

    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getEventName()
    {
        return $this->eventName;
    }
    public function setEventName($eventName)
    {
        $this->eventName = $eventName;
    }

    public function getImageUrl()
    {
        return $this->imageUrl;
    }
    public function setImageUrl($imageUrl)
    {
        $this->imageUrl = $imageUrl;
    }

    public function getPerformanceDate()
    {
        return $this->performanceDate;
    }
    public function setPerformanceDate($performanceDate)
    {
        $this->performanceDate = $performanceDate;
    }

    public function getAreaName()
    {
        return $this->areaName;
    }
    public function setAreaName($areaName)
    {
        $this->areaName = $areaName;
    }

    public function getSeats()
    {
        return $this->seats;
    }
    public function setSeats($seats)
    {
        $this->seats = $seats;
    }

    public function getTickets()
    {
        return $this->tickets;
    }
    public function setTickets($tickets)
    {
        $this->tickets = $tickets;
    }

    public function getVenueId()
    {
        return $this->venueId;
    }
    public function setVenueId($venueId)
    {
        $this->venueId = $venueId;
    }

    public function getVenueName()
    {
        return $this->venueName;
    }
    public function setVenueName($venueName)
    {
        $this->venueName = $venueName;
    }

    public function getVenueAddress()
    {
        return $this->venueAddress;
    }
    public function setVenueAddress($venueAddress)
    {
        $this->venueAddress = $venueAddress;
    }

    public function getVenueCity()
    {
        return $this->venueCity;
    }
    public function setVenueCity($venueCity)
    {
        $this->venueCity = $venueCity;
    }

    public function getVenuePostCode()
    {
        return $this->venuePostCode;
    }
    public function setVenuePostCode($venuePostCode)
    {
        $this->venuePostCode = $venuePostCode;
    }

    public function getPriceSubTotal()
    {
        return $this->priceSubTotal;
    }
    public function setPriceSubTotal($priceSubTotal)
    {
        $this->priceSubTotal = $priceSubTotal;
    }

    public function getReservationValidity()
    {
        return $this->reservationValidity;
    }
    public function setReservationValidity($reservationValidity)
    {
        $this->reservationValidity = $reservationValidity;
    }

    public function getExpirationDate()
    {
        return $this->expirationDate;
    }
    public function setExpirationDate($expirationDate)
    {
        $this->expirationDate = $expirationDate;
    }
}

